test/e2e/steps/login.steps.js

```diff
@@ -0,0 +1,18 @@
+const LoginAssert = require('../asserts/Login.assert');
+const LoginPage = require('../pages/LoginPage');
+const { Given, When, Then } = require('cucumber');
+
+Given(/^user navigates to "([^"]*)"$/, (endpoint) => {
```

Comment:
According the article https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/ Given step should be a status, not an action

Reply:
yep, totally agree, please report it as a bug, this code was defined before the movements in steps definitions. In this case, this step was defined by @R0m3rCh